WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -81,11 +81,13 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v22.8-whats-new"
+msgctxt "v22.9-whats-new"
 msgid ""
-"We disabled blogging reminders for self-hosted sites that aren’t connected to Jetpack, since this feature is only available in the Jetpack app.\n"
+"We made some big changes to the block editor, including styles, colors, and icons. You’ll also find more buttons and block options in the header toolbar, including the undo and redo buttons. That’s right, you now have more tools than Batman’s utility belt.\n"
```

Comment:
I love Grace's humorous tone in our release notes 😄  🦇

Reply:
Apart from being funny in its own right, it gives the release notes a playful personality. I really like it.